server/src/server.rs

```diff
@@ -255,17 +263,16 @@ async fn keys_check(
 }
 
 #[derive(Clone)]
-pub struct ServerState {
+pub(crate) struct ServerState {
```

Comment:
Do we want this to be pub? Shouldn't it only be the "new" function which is public, since we dont want it to be initialized without a call to "new"